assignments-list/snippet.html

```diff
@@ -0,0 +1,54 @@
+
+
+<script>
+var $assignmentsList = $("<div>", {
+                id: Frog.Utilities.generateSimpleId()
+              }
+    );
+    $assignmentsList.html(
+       '<b>Your open assignments:</b>'+
+       '<table class="table" id="your_assignments">'+
+            '<tr>'+
+            '<th>Subject</th>'+
+            '<th>Title</th>'+
+            '<th>Due Date</th>'+
+            '<th>Instructions</th>'+
+            '</tr>'+
+       '</table>'
+    );
+
+
+
+Frog.Model
+    .api('assignmentspupils.getmyassigned', {
```

Comment:
assignmentpupils.getmyassigned is about to be deprecated and removed from the codebase.